build.gradle

```diff
@@ -40,6 +40,7 @@ configurations {
 }
 
 dependencies {
+  compile group: 'io.github.bonigarcia', name: 'webdrivermanager', version: '1.6.0'
```

Comment:
It will be nice to exclude transitive dependencies that already exists in project dependencies.
e.g. com.google.code.gson
Some projects doesn't not accept to have different jar's version in classpath.